### DISCUSSION TITLE
Is it possible to use Pandas in React Native?

### DISCUSSION THREAD

**Discussion by GollyJer** [chakra-ui/panda#745] [Q&A] [answered]
Is it possible to use Panda in React Native?
Looks very promising but I want to keep using an approach that works for our native and web projects.

**Comment by anubra266** [comment, marked as answer]
Hi @GollyJer That's not possible yet

**Reply by drianoaz** [reply]
Are there any plans to support it in the future?

**Reply by djMax** [reply]
We would be willing to contribute money and time to this effort, any philosophical thoughts about it?

**Reply by jeffscottward** [reply]
https://www.dripsy.xyz

This is more a cousin to ThemeUI which is based on emotion which is still trying to figure out RSC support as of 10/28/2023.

I've been using Solito w/ Dripsy to "cover all bases" in one repo.

https://github.com/nandorojo/solito

Give that PandaCSS is meant to be static and has a lot of flexibility, I think one way or another it could support RN somehow but would mean a particularly focused effort.

Hope this helps get the gears turning. 

EDIT:
Given RN uses straight up style objects 
https://reactnative.dev/docs/style

Perhaps just pulling in tokens with the functions is enough?
https://panda-css.com/docs/theming/usage#style-attribute
